build_library/toolchain_util.sh

```diff
@@ -38,7 +43,7 @@ declare -r \
 # Usage: get_portage_arch chost
 get_portage_arch() {
     case "$1" in
-        aarch64*)   echo arm;;
+        aarch64*|arm64*) echo arm64;;
```

Comment:
Is arm64 a valid CHOST prefix? The list that this was copied from (which has since been updated for arm64 in Gentoo) doesn't include it:
```sh
case ${CTARGET} in
                aarch64*)   TARCH=arm64;;
                alpha*)     TARCH=alpha;;
                arm*)       TARCH=arm;;
                hppa*)      TARCH=hppa;;
                ia64*)      TARCH=ia64;;
                i?86*)      TARCH=x86;;
...
```